README.md

```diff
@@ -156,6 +156,17 @@ Learn more about <a href="https://meshery.layer5.io/docs/architecture">Meshery's
 <a name="contributing"></a><a name="community"></a>
 Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Meshery Contributors Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>!
 
+<p style="clear:both;">
+<a href ="https://layer5.io/community"><img alt="MeshMates" src="docs/assets/images/Layer5-MeshMentors.png" style="margin-right:10px; margin-bottom:7px;" width="28%" align="left" /></a>
+<h3>Find your MeshMate</h3>
+
+MeshMates are experienced Layer5 community members, who will help you learn your way around, discover live projects and expand your community network. 
+<p>Become a <b>Meshtee</b> today!</p>
+
+Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 community page</a>. <br />
```

Comment:
Whoa. Well done, @chandrashritii!

A small tweak - could we drop "page" and say `Find out more in the <a href="https://layer5.io/community#meshmate">Layer5 community</a>.`